src/app.tsx

```diff
@@ -16,7 +16,23 @@ export default function App() {
 				<MetaProvider>
 					<Title>Solid Docs</Title>
 					<ErrorBoundary
-						fallback={(e) => <pre>{JSON.stringify(e, null, 2)}</pre>}
+						fallback={(e) => {
+							console.error(e);
+							return (
+								<>
+									<Title>404 - SolidDocs</Title>
+									<Layout isError={!!e}>
```

Comment:
```suggestion
									<Layout isError={Boolean(e)}>
```

my nitpick against implicit programming. Being explicit makes code more readable and clear.